code/modules/reagents/reagent_containers/cups/_cup.dm

```diff
@@ -133,6 +133,7 @@
 			return ITEM_INTERACT_BLOCKING
 
 		var/trans = target.reagents.trans_to(src, amount_per_transfer_from_this, transferred_by = user)
+		playsound(target.loc, pick('sound/effects/liquid_pour1.ogg', 'sound/effects/liquid_pour2.ogg', 'sound/effects/liquid_pour3.ogg'), 50)
```

Comment:
this should really be an sfx define... sigh

Reply:
Sure thing [66f7310](https://github.com/tgstation/tgstation/pull/86859/commits/66f7310466034b252bd8010ab5f493fa9417011f)